backend/scripts/seed_data.py

```diff
@@ -283,6 +286,24 @@
     ),
 ]
 
+TEST_CASE_ISSUE_REQUESTS = [
+    ReportedIssueRequest(
+        template_id=END_TEST_EXECUTION_REQUESTS[0].test_results[0].name,
```

Comment:
Probably this wont make any difference as it is a seed data script, but I think this should be either `case_name=...` or `template_id=(...).template_id`